migration/sql-test-files/081-query-null-title.sql

```diff
@@ -0,0 +1,9 @@
+DELETE FROM spaces where id='171d52ff-fa00-46d7-ac24-94269908ad7a';
+INSERT INTO spaces (id, name) VALUES ('171d52ff-fa00-46d7-ac24-94269908ad7a', 'test space');
+
+-- empty query name
```

Comment:
In this test, the name is not empty as in `''` but it is `NULL`. Therefore you're testing that Postgres does it job when checking for `NOT NULL` values. I think we don't need to go this deep.

Reply:
yes, hence the filename. and added https://github.com/fabric8-services/fabric8-wit/pull/1824/commits/8936aa1ff1018e648fff1a0e66be3355a8eef093#diff-06f8f427154426f3c99f95b5424025c7 to see the empty title
do you suggest I should not have this test NULL title?

Reply:
yes

Reply:
the empty title is okay but we should rely on postgres to check that the title cannot be null. Also, if there's a postgres check on the title that it cannot bet `''`, then I think you can get rid of that test for the empty title alltogether. WDYT?

Reply:
sure, I will get rid off null_title test thanks!